booking-service/src/test/java/co/axelrod/bookingservice/service/BookingServiceTest.java

```diff
@@ -88,6 +120,45 @@ void happyPathWithPaymentIssues() {
             - Booking canceled
             """)
     void paymentIssuesBookingDeclined() {
+        UUID stateMachineUUID = bookingService.createBooking(firstBooking);
+        bookingService.confirmBooking(stateMachineUUID);
+        bookingService.handlePaymentFailure(stateMachineUUID);
+        bookingService.cancelBookingAfterPaymentFailure(stateMachineUUID);
+
+        var stateMachine = bookingService.getBooking(stateMachineUUID);
+
+        Assertions.assertEquals(States.CANCELED, stateMachine.getState().getId());
+    }
+
+    @Test
+    @DisplayName("""
+            Booking canceled - :
+            - Guest creates booking
+            - Guest cancels booking
+            """)
+    void customerSelfBookingDeclined() {
+        UUID stateMachineUUID = bookingService.createBooking(firstBooking);
+        bookingService.cancelBookingByCustomer(stateMachineUUID);
```

Comment:
`bookingService.cancelBookingByCustomer` already returned the status `States.CANCELED` for the next assertion.